kong/plugins/datadog/handler.lua

```diff
@@ -112,12 +109,12 @@ local function log(premature, conf, message)
   end
 
   local stat_name  = {
-    request_size     = name .. ".request.size",
-    response_size    = name .. ".response.size",
-    latency          = name .. ".latency",
-    upstream_latency = name .. ".upstream_latency",
-    kong_latency     = name .. ".kong_latency",
-    request_count    = name .. ".request.count",
+    request_size     = "api.request.size",
+    response_size    = "api.response.size",
+    latency          = "api.latency",
+    upstream_latency = "api.upstream_latency",
+    kong_latency     = "api.kong_latency",
+    request_count    = "api.request.count",
```

Comment:
This 0.13.0, APIs are deprecated in favor of Services and Routes. Why not using a more generic name?

Reply:
Some re-alignment is necessary before merging this, thanks for the feedback.  At this point 'api' is just a namespace difference to denote the difference between data gathered from the stats endpoint by Datadog's agent (that don't have this extra 'api' namespace, and data that is pushed via statsd with this plugin.  This could easily be "statsd" to show the user that this is data pushed from statsd, or this could be "service" or "route" if that makes it make more sense.  Thoughts?